test/parallel/test-http-server-keep-alive-timeout.js

```diff
@@ -1,77 +1,122 @@
 'use strict';
 
-const common = require('../common');
-const assert = require('assert');
-const http = require('http');
-const net = require('net');
+module.exports.testServerKeepAliveTimeout = function(common, isHttps) {
+  let http;
+  let net;
+  let createOptions;
+  let serverOptions;
+  let createServer;
 
-const tests = [];
+  if (isHttps) {
```

Comment:
Since this is the only `if` id do this separately for each file, and pass all the objects as args.